applications/visualizer/frontend/src/components/viewers/ThreeD/CustomFormControlLabel.tsx

```diff
@@ -3,13 +3,21 @@ import { Box, FormControlLabel, Stack, Tooltip, Typography } from "@mui/material
 import { vars } from "../../../theme/variables.ts"; // Adjust the import path as needed
 import CustomSwitch from "../../ViewerContainer/CustomSwitch.tsx";
 
+interface CustomFormControlLabel {
+  label: React.ReactNode;
+  tooltipTitle: React.ReactNode;
+  helpText: React.ReactNode;
+  checked?: boolean;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => void;
+}
+
 const { gray50, gray600, gray400B } = vars;
-const CustomFormControlLabel = ({ label, tooltipTitle, helpText }) => {
+const CustomFormControlLabel = ({ label, tooltipTitle, helpText, checked, onChange }: CustomFormControlLabel) => {
```

Comment:
Should we move this somewhere else? It's imported by the EM viewer.
I can also just copy it to the EM viewer.